integrations/micrometer/micrometer/src/main/java/module-info.java

```diff
@@ -32,17 +32,17 @@
 
     requires java.logging;
 
-    requires static jakarta.annotation;
+    requires jakarta.annotation;
```

Comment:
This module seems to be for Helidon SE, and so `jakarta.annotation` may not be on classpath.

Reply:
ok